### PR TITLE
Bug 2216707: Fixes for topology view for external mode

### PR DESCRIPTION
**Pull request by bipuladh**
Screenshot from 2023-06-15 19-08-19

### CODE REVIEW

File: locales/en/plugin__odf-console.json [hunk 1]
```diff
@@ -843,7 +843,9 @@
   "Expand to fullscreen": "Expand to fullscreen",
   "Exit fullscreen": "Exit fullscreen",
   "Back to main view": "Back to main view",
+  "Topology view is not supported for External Mode": "Topology view is not supported for External Mode",
   "No StorageCluster found": "No StorageCluster found",
+  "Topology view is not supported for external mode": "Topology view is not supported for external mode",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:848]: same text twice ?? with "E" and "e" in external

File: packages/odf/components/topology/Topology.tsx [hunk 2]
```diff
@@ -575,7 +579,13 @@ const Topology: React.FC = () => {
   );
 };
 
-const TopologyViewEmptyState: React.FC = () => {
+type TopologyViewEmptyStateProps = {
+  isExternalMode?: boolean;
+};
+
+const TopologyViewErrorMessage: React.FC<TopologyViewEmptyStateProps> = ({
```
Comment (debjyoti-pandit) [inline packages/odf/components/topology/Topology.tsx:586]: shouldn't here the component name and the props type name be the same?

File: packages/odf/components/topology/Topology.tsx [hunk 3]
```diff
@@ -597,23 +607,34 @@ const TopologyViewEmptyState: React.FC = () => {
...
+        {isExternalMode
+          ? t('Topology view is not supported for External Mode')
+          : t('No StorageCluster found')}
       </Title>
       <EmptyStateBody>
-        {t('Set up a storage cluster to view the topology')}
+        {isExternalMode
+          ? t('Topology view is not supported for external mode')
+          : t('Set up a storage cluster to view the topology')}
       </EmptyStateBody>
```
Comment (SanjalKatiyar) [inline packages/odf/components/topology/Topology.tsx:618]: same title and description for external mode ??
also, "E" and "e" in external in both texts...

**Pull request merged**